src/main/python/deep_qa/layers/backend/max.py

```diff
@@ -0,0 +1,41 @@
+from keras import backend as K
+from keras.layers import Layer
+from overrides import overrides
+
+from ...tensors.backend import switch, very_negative_like
+
+
+class Max(Layer):
+    '''
```

Comment:
i'm going to start being a lot more hard-nosed about docstring formatting since we actually have docs to maintain now --- [PEP257](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) specifies that these should be triple quotes.